lochness/tree/__init__.py

```diff
@@ -97,7 +97,7 @@ def get(data_type, base, **kwargs):
             sub_folder = Templates[data_type]['raw'].substitute(
                     base='', **kwargs)[1:]
             sub_folder = re.sub('/(raw|processed)', '', sub_folder)
-            raw_folder = base / sub_folder / phoenix_id
+            raw_folder = base / phoenix_id / sub_folder
```

Comment:
major change

Reply:
Isn't `phoenix_id` same as `sub_folder`? If not, can you post a tree so we know where `raw_folder` and `processed_folder` reside?

Reply:
> study / raw / datatype / subject -> fixed to -> study / raw / subject / data

Sorry, just saw your top description but isn't it contingent upon using `--bids`? Or is RPMS organizing according to BIDS by default?

Reply:
>Sorry, just saw your top description but isn't it contingent upon using --bids? Or is RPMS organizing according to BIDS by default?

The plan is to use BIDS by default for both Pronet and Prescient

Reply:
> Isn't phoenix_id same as sub_folder? If not, can you post a tree so we know where raw_folder and processed_folder reside?

See the figure below
![image](https://user-images.githubusercontent.com/3075783/126072222-1f165c69-77a3-4117-8e3f-7f65d71aea33.png)

Reply:
Thank you @kcho, I understand that `BIDS: True` corresponds to the right branch. But can you point out which directories are `base`, `phoenix_id`, and `sub_folder`:

> raw_folder = base / phoenix_id / sub_folder

Reply:
eg)
base = `study / raw` or `study/processed`
phoenix_id = `subject01`
sub_folder = `actigraphy`

Reply:
Okay got it. I guess I am used to interpreting `sub_folder` as a folder with subject ID. That created the confusion. A more appropriate name could be `data_folder` but I won't insist on it.